OrchardCore.Commerce/Models/ShoppingCart.cs

```diff
@@ -3,121 +3,126 @@
-            }
-            else
-            {
-                Items.Add(item);
-            }
+            var existingItem = Items[existingIndex];
+            Items[existingIndex] = existingItem.WithQuantity(existingItem.Quantity + item.Quantity);
         }
+        else
+        {
+            Items.Add(item);
+        }
+    }
 
-        /// <summary>
-        /// Removes an item from the cart.
-        /// </summary>
-        /// <param name="item">The product variant to remove. Quantity will be ignored.</param>
-        public void RemoveItem(ShoppingCartItem item)
+    /// <summary>
+    /// Removes an item from the cart.
+    /// </summary>
+    /// <param name="item">The product variant to remove. Quantity will be ignored.</param>
+    public void RemoveItem(ShoppingCartItem item)
+    {
+        var existingIndex = IndexOf(item);
+        if (existingIndex != -1)
         {
-            int existingIndex = IndexOf(item);
-            if (existingIndex != -1)
-            {
-                Items.Remove(Items[existingIndex]);
-            }
+            Items.Remove(Items[existingIndex]);
         }
+    }
 
-        /// <summary>
-        /// Sets the list of prices for the item.
-        /// </summary>
-        /// <param name="item">The item.</param>
-        /// <param name="prices">The list of prices.</param>
-        public void SetPrices(ShoppingCartItem item, IEnumerable<PrioritizedPrice> prices)
+    /// <summary>
+    /// Sets the list of prices for the item.
+    /// </summary>
+    /// <param name="item">The item.</param>
+    /// <param name="prices">The list of prices.</param>
+    public void SetPrices(ShoppingCartItem item, IEnumerable<PrioritizedPrice> prices)
+    {
+        var existingIndex = IndexOf(item);
+        if (existingIndex != -1)
+        {
+            Items.Remove(Items[existingIndex]);
+            Items.Insert(existingIndex, item.WithPrices(prices));
+        }
+        else
         {
-            int existingIndex = IndexOf(item);
-            if (existingIndex != -1)
-            {
-                Items.Remove(Items[existingIndex]);
-                Items.Insert(existingIndex, item.WithPrices(prices));
-            }
-            else throw new InvalidOperationException("Can't set prices on a product that's not in the cart.");
+            throw new InvalidOperationException($"Can't set {nameof(prices)} on a product that's not in the cart.");
         }
+    }
 
-        /// <summary>
-        /// Finds the index of the first line in the cart with the same product variant as the passed in item (quantity may be different).
-        /// </summary>
-        /// <param name="item">The item to find.</param>
-        /// <returns>The index of the item, or -1 if not found.</returns>
-        private int IndexOf(ShoppingCartItem item)
+    /// <summary>
+    /// Finds the index of the first line in the cart with the same product variant as the passed in item (quantity may be different).
```

Comment:
I have went ahead and reviewed every multiline summaries (in case you are wondering `summary>\n(\s*///[^\n]*\n){2,}\s*///\s*</summary>`) to make sure there is nothing else left to rewrap.

Reply:
Thank you sooo much! 😁